### ISSUE TITLE
GraXpert fails to start with unusual desktop sizes

### ISSUE THREAD

**Issue by garin1000** [Steffenhir/GraXpert#73]
Hi all,
when trying to start GraXpert on a 11:9 (1760x1440 pixels) display, it fails to load, and shows the splash screen and an empty windows indefinitely long. It works fine on the same system with a 32:9 display.
 
OS is Windows 11, GraXpert is Dione RC-5 (v1.0.4)

GraXpert has three processes open, one of which is inactive. One of the active processes uses about 5% CPU.

![image](https://user-images.githubusercontent.com/37046101/221517547-bad67391-4494-46d6-978f-aa0ec5e97b56.png)

Closing the application windows works, but the multiprocessing fork process remains active.

Would be great if you could have a look at it.

Cheers
Norbert

**Comment by wx273** [comment]
I have the identical issue.
Splash banner on startup then it freezes.
Same version of GraXpert.
Win 10 Home 22H2 and display set at 1920x1080

**Comment by Steffenhir** [comment]
Thank you for reporting the issue! We will take a look at how we scale the user interface. I suspect that the problem arises in the detection of the display dpi.

**Comment by Steffenhir** [comment]
@wx273 @garin1000 Could you try if the problem still persists with the most recent release?

**Comment by galacticfungus** [comment]
Hi, for me I experienced this problem with Kaspersky antivirus, adding an exclusion did not help until I allowed Graxpert access to the Kaspersky antivirus interface. This makes no sense but until I did that Kaspersky would scan the executable constantly, checking all the python scripts. This was the free version of Kaspersky antivirus; I include the log file, but I don't think there is anything relevant.
[graxpert.log](https://github.com/Steffenhir/GraXpert/files/12377161/graxpert.log)
